plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -131,25 +131,29 @@ export class GithubPublisher implements PublisherBase {
 
     const { data } = await repoCreationPromise;
 
-    if (access?.startsWith(`${owner}/`)) {
-      const [, team] = access.split('/');
-      await client.teams.addOrUpdateRepoPermissionsInOrg({
-        org: owner,
-        team_slug: team,
-        owner,
-        repo: name,
-        permission: 'admin',
-      });
-      // no need to add access if it's the person who own's the personal account
-    } else if (access && access !== owner) {
-      await client.repos.addCollaborator({
-        owner,
-        repo: name,
-        username: access,
-        permission: 'admin',
-      });
+    try {
+      if (access?.startsWith(`${owner}/`)) {
+        const [, team] = access.split('/');
+        await client.teams.addOrUpdateRepoPermissionsInOrg({
+          org: owner,
+          team_slug: team,
+          owner,
+          repo: name,
+          permission: 'admin',
+        });
+        // no need to add access if it's the person who owns the personal account
+      } else if (access && access !== owner) {
+        await client.repos.addCollaborator({
+          owner,
+          repo: name,
+          username: access,
+          permission: 'admin',
+        });
+      }
+    } catch (e) {
+      e.message = `Failed to add access to '${access}': ${e.message}`;
```

Comment:
This will have weird consequences.

![Screenshot 2021-03-10 at 13 51 36](https://user-images.githubusercontent.com/3097461/110632333-c9cb5680-81a7-11eb-8cce-04aedd8ed785.png)

You probably want to ```throw new Error(`More text, ${e.message}`)```

Reply:
haha, I'll update :)